### ISSUE TITLE
18Korea

### ISSUE THREAD

**Issue by fcboyd** [tobymao/18xx#11368]
**What's your idea?**

Is there anyone interested in taking on 18 Korea. I know that a steam edition of the game exists, but I think it would be much easier to get players together for a game if it was implemented on .games. 

High-level checklist of changes:

- [ ] lots of asset cards, some fundamentally change rules in the game
- [ ] Korean War - Eliminates North Korea and any corporations that don't have a token in South Korea (triggers on purchase of final 4-Train)
- [ ] Outer Stations - Can build home stations above and beyond available station tokens in that era (i.e. do not need to reserve station slot for companies starting there).
- [ ] Must have valid route between corporations to perform cross-company train purchase
- [ ] Running multiple express trains uses the value of the lowest express train route.

Let us know your proposed changes.

**Are there related ideas?**

Other contributors may be working on something similar, [check out the list of open feature requests](https://github.com/tobymao/18xx/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) to see if there's opportunity to collaborate.

**Did you join the Slack?**

Yes

Check out `#18xxgamesdev` [on the slack](https://join.slack.com/t/18xxgames/shared_invite/zt-27imtsj2u-vussFAqtecmACsycjdsIhg) and meet your fellow contributors!

**Comment by slapphappyjazz** [comment]
I believe the "running multiple express trains" is already implemented for 18NewEngland.
The restriction on cross-company train purchase is also needed for 18Royal Gorge (I submitted a bug report about that earlier this evening).
The fundamental question is whether you have the permission of the designer and the publisher.

**Comment by fcboyd** [comment]
Yeah, I should have clarified that I have no information on whether permission has been granted or denied. I sent out an email to the publisher floating the question but have yet to receive a response.

**Comment by ollybh** [comment]
There is a spreadsheet which shows which games we have been given, or refused, permission to implement on 18xx.games: https://docs.google.com/spreadsheets/d/1ndWr805MZ8pIzSgh94bb6C3zr3Noh4Xpc3Gl2FN_-6g/edit#gid=0

18Korea isn't listed, so nobody would be allowed to start development at the moment.

@fcboyd Let us know if you hear back from the publisher so we can update this list.

**Comment by fcboyd** [comment]
Will do, I wanted to get this request submitted as a mechanism for updating the permission spreadsheet either way.

**Comment by michaeljb** [comment]
> Must have valid route between corporations to perform cross-company train purchase

See #11386

> lots of asset cards, some fundamentally change rules in the game

There are so many of these, I expect this will easily be the number one time sink if anyone takes this game on.

Since there's a random setup component, I would recommend implementing subsets of these at a time, and they could be grouped together via optional rule. That way, when a new one is added, the shuffling with the newly-expanded deck doesn't automatically break all previous games.

There should also be an optional rule to use none of them, and there should be enforcement to select some optional rules to pick which assets to play with. Once they're all implemented, then the optional rules could be hidden and the default could be to play with them all. This would prevent games from constantly being broken just from progress being made on the assets deck.

**Comment by fcboyd** [comment]
Have tried reaching out via email + social media (Instagram) to the publisher and have not heard back either way.

**Comment by Ticketautomat1** [comment]
I have contacted the designer and he has denied to publish 18Korea on this platform. Somebody can close this thread.
